### PR TITLE
feat: support setup.sh and modify readme.md

### PR DESCRIPTION
**Pull request by artikell**

### CODE REVIEW

**Comment by tommyme** [comment]
There are some tips can help you develop bmx plugin, after you've installed bmx plugin, you can open Quick Run view with command palette `>focus on quick run view`.

There are some tasks that are predefined in it:
1. rollup build watch -- generate package.json, build for production, watch change.
2. tsc build -- build for debug, watch change.
3. Run Extension.

There are some scripts:
1. use `node rollup.config.mjs` -- to generate package.json


In addition, in vscode plugin project, we must use yarn to build the package and upload it to vscode market. So we don't use npm to install dependencies.

**Comment by artikell** [comment]
> There are some tips can help you develop bmx plugin, after you've installed bmx plugin, you can open Quick Run view with command palette `>focus on quick run view`.
> 
> There are some tasks that are predefined in it:
> 
> 1. rollup build watch -- generate package.json, build for production, watch change.
> 2. tsc build -- build for debug, watch change.
> 3. Run Extension.
> 
> There are some scripts:
> 
> 1. use `node rollup.config.mjs` -- to generate package.json
> 
> In addition, in vscode plugin project, we must use yarn to build the package and upload it to vscode market. So we don't use npm to install dependencies.
> 
> Additions to the document are welcomed❤.

When I wanted to develop, rollup and packagejson got stuck in a circular dependency. So, I still believe that there needs to be a script to help everyone complete the initialization of the environment.

**Comment by tommyme** [comment]
I think the solution to the diffculty of dev startup is to make dev document better.
Although a `setup.sh` can make thing in one step, windows developers can't benefit from it.
And also, a `setup.sh` and a `setup.cmd` would make project root dir dirty.

File: rollup.config.mjs
Comment (tommyme) [inline rollup.config.mjs:1]: no need to change

**Pull request closed without merging**